bootstrap.php

```diff
@@ -43,9 +42,10 @@
 // Load Composer autoloader:
 require_once(dirname(__FILE__) . '/vendor/autoload.php');
 
+\PHPCI\ErrorHandler::register();
+
 if (defined('PHPCI_IS_CONSOLE') && PHPCI_IS_CONSOLE) {
     $loggerConfig = LoggerConfig::newFromFile(__DIR__ . "/loggerconfig.php");
```

Comment:
The $loggerConfig isn't used in bootstrap.php but in both the console and daemonise scripts. We can 1) let these scripts as they are, 2) duplicate this piece of code in the two scripts, 3) merge daemonise into console and move this piece of code there.